src/Controls/src/Core/Menu/MenuItem.cs

```diff
@@ -100,27 +96,6 @@ public bool IsEnabled
 			set => SetValue(IsEnabledProperty, value);
 		}
 
-		/// <include file="../../docs/Microsoft.Maui.Controls/MenuItem.xml" path="//Member[@MemberName='StyleClass']/Docs/*" />
-		[System.ComponentModel.TypeConverter(typeof(ListStringTypeConverter))]
-		public IList<string> StyleClass
```

Comment:
This will have to stay unfortunately as it will break API. However, you can do a `public new IList<string> StyleClass` and just call base in there. Same for @class,

Reply:
Chatted with @mattleibow offline and he validated we're ok to remove these without breaking ABI compatibility

Reply:
I was wrong. TIL, or rather YIL since it was yseterday when I found out.